paddle/framework/op_registry.h

```diff
@@ -143,28 +131,22 @@ class OpRegistry {
     OpProto& op_proto = protos_[op_type];
```

Comment:
The creators_ map's key is not op_type, but op_type+device_type+(data_type).
Here, we may consider data_type later(only float is enough for now)